java/clients/src/test/java/sleeper/clients/admin/InstanceConfigurationScreenTest.java

```diff
@@ -363,15 +359,8 @@ void shouldShowValidationFailuresForMultipleProperties() throws Exception {
                     "Unset before, default value: 900\n" +
                     "After (not valid, please change): abc\n" +
                     "\n" +
-                    "sleeper.default.fs.s3a.readahead.range\n" +
-                    "The readahead range set on the Hadoop configuration when reading Parquet files in a query\n" +
-                    "(see https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html).\n" +
-                    "Unset before, default value: 64K\n" +
-                    "After (not valid, please change): def\n" +
-                    "\n" +
                     "Found invalid properties:\n" +
                     "sleeper.compaction.job.creation.timeout.seconds\n" +
-                    "sleeper.default.fs.s3a.readahead.range\n" +
```

Comment:
The test isn't valid after this change. Its purpose is to show we can display multiple validation failures when more than one property failed to validate. It's only changing one property now.